flask_appbuilder/security/manager.py

```diff
@@ -701,6 +713,48 @@ def _decode_and_validate_azure_jwt(self, id_token: str) -> Dict[str, str]:
 
         return jwt.decode(id_token, options={"verify_signature": False})
 
+    def _get_authentik_jwks(self, jwks_url) -> dict:
+        import requests
+
+        resp = requests.get(jwks_url)
+        if resp.status_code == 200:
+            return resp.json()
+        return False
```

Comment:
A bit late to the game but I am incorporating latest FAB changes to Airflow. I guess this should be a {} rather than False (or the return type should be `dict | bool`  (but empty dict is likely better as it is Falsey value anyway.

Reply:
...
You are right.

I don't know how I missed that 😧